sklearn/linear_model/tests/test_logistic.py

```diff
@@ -875,3 +875,11 @@ def test_warm_start():
                         assert_greater(2.0, cum_diff, msg)
                     else:
                         assert_greater(cum_diff, 2.0, msg)
+
+
+@ignore_warnings
```

Comment:
`auto`  is deprecated in favor of `balanced`? In that case put a please use `assert_warns`  to make it expliclit that we expect a `DeprecationWarning`.